services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_tasks.py

```diff
@@ -103,9 +108,6 @@ async def upsert_tasks_from_project(
                 node_extras: ServiceExtras = await director_client.get_service_extras(
                     service_key_version
                 )
-            if not node_details:
-                continue
-
```

Comment:
why is this one gone? bad merge?

Reply:
my fault was not able to merge it correctly. Will add it back.